dbt_project.yml

```diff
@@ -54,6 +54,7 @@ models:
 
 vars:
   dbt_date:time_zone: Etc/UTC
+  fullstory_incremental_interval: INTERVAL 7 DAY
```

Comment:
This is a pretty large default interval. We could probably get away with 2 days or 30 hours.